.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+# Contributing
+
+When contributing to this repository, please first create an issue and link the PR with it.
+
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+## Pull Request Process
+
+1. Update the README.md with details of changes including example hcl blocks and [example files](./examples) if appropriate.
```

Comment:
this point can be removed as the contributor does not need to edit the main README.md file in the root directory and the README.md in the charts dir will be updated automatically by pre-commit/helm-docs